src/lib/style/components/_cards.scss

```diff
@@ -125,9 +144,11 @@ $text-color-lighter: #898989;
 
   .SRC-cardMetadata {
     display: block;
-    background-color: $color-gray-background;
-    margin-top: $baseline-grid;
-    padding: $baseline-grid $baseline-grid $baseline-grid 15%;
```

Comment:
Now only applies the padding-left of 15% if the card is showing an icon (not an image)